src/main/java/application/UserInteractor.java

```diff
@@ -6,22 +6,10 @@
  * Requires simply that an implementing class stores a User object and accepts requests.
  */
 public abstract class UserInteractor implements InputBoundary{
-    private final User user;
+    private User user;
 
     /**
      * Construct a new UserInteractor object to interact with a User.
-     * @param user  User to interact with
      */
-    public UserInteractor(User user) {
```

Comment:
Similar to the above comment, since it's an abstract class I don't think it needs this constructor either.